python/docs/conf.py

```diff
@@ -47,17 +47,17 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'GraphFrames'
-copyright = u''
+project = u'pygraphframes'
+copyright = u'2016, Author'
```

Comment:
Then I think it is more customary to put something like "2016, the
GraphFrames project authors" and list the contributors in the README. This
is a nit, though, we can change it later.

On Fri, Feb 19, 2016 at 4:25 PM, jkbradley <notifications@github.com> wrote:

> In python/docs/conf.py
> <https://github.com/graphframes/graphframes/pull/10#discussion_r53537393>:
>
> > @@ -47,17 +47,17 @@
> >  master_doc = 'index'
> >
> >  # General information about the project.
> > -project = u'GraphFrames'
> > -copyright = u''
> > +project = u'pygraphframes'
> > +copyright = u'2016, Author'
>
> But also Ankur, Feynman, Xiangrui, and hopefully others in the future. I'd
> prefer to leave it generic.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/graphframes/graphframes/pull/10/files#r53537393>.
>